src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/FeatureToggleService.java

```diff
@@ -43,4 +46,19 @@ public boolean isHwfSuccessfulNotificationLetterEnabled() {
     public boolean isAssignedToJudgeNotificationLetterEnabled() {
         return isFeatureEnabled(ASSIGNED_TO_JUDGE_NOTIFICATION_LETTER);
     }
+
+    /**
+     * Given runtime feature toggle status, returns fields that should be ignored during serialisation (i.e. not
+     * serialised to JSON)
+     * @return a map with Class of ignored fields as key and field names as value
+     */
+    public Map<Class, List<String>> getFieldsIgnoredDuringSerialisation() {
+        Map<Class, List<String>> ignoredFields = Maps.newHashMap();
+
+        if (!isApprovedConsentOrderNotificationLetterEnabled()) {
+            ignoredFields.put(ApprovedOrder.class, asList("consentOrderApprovedNotificationLetter"));
```

Comment:
Hello @jakub-kozlowski. As per our conversation, can you make this a constant and explain in a comment that this has to match the fields that are in the "non-prod" file in CCD definitions?